index.html

```diff
@@ -1425,6 +1426,15 @@ <h2>
                 constraint is applied (<code>true</code>) or not (<code>false</code>).
               </p>
             </dd>
+            <dt>
+               <dfn>suppressLocalAudioPlayback</dfn> of type {{boolean}}
+            </dt>
+            <dd>
+              <p>
+                Indicates whether the <a href="#dfn-suppresslocalaudioplayback">suppressLocalAudioPlayback</a>
+                constraint is applied (<code>true</code>) or not (<code>false</code>).
```

Comment:
I think we should just use the same language as [here](https://w3c.github.io/mediacapture-screen-share/#dfn-suppresslocalaudioplayback):
```suggestion
                Indicates whether or not the user agent is applying
                [=local audio playback suppression=] to the source.
```
The non-compliance option happens at a lower level that isn't observable. Readers can easily click on [local audio playback suppression](https://w3c.github.io/mediacapture-screen-share/#dfn-local-audio-playback-suppression) to reveal that part.